src/ex1/style.css

```diff
@@ -0,0 +1,4 @@
+@import "styles/common.css";
+@import "styles/buttons.css";
+@import "styles/classes.css";
```

Comment:
It's amazing that you separated the styles into different modules, however, I think the separating logic could be improved, here are a few notes:
1. Separating between classes and ids could be counter-productive as in most cases we want to separate between different components and areas so we could focus only on what's affecting the area we are working on.
2. common is often used for common styles used in many areas, for example .button, .hidden, .centered etc..

I would separate this into 3 modules:
- app.css - baseline of the app - ex: HTML, body etc..,
- common.css - Reusable styles that we can use across the app, but also the baselines for different core elements (button/input)
- todo.css - style the todo component

Reply:
I created app.css, common.css and todo.css as you said but also kept button.css in a separate file because I feel it is more convenient for me right now